packages/ui/src/ui/components/ColumnSelector/ColumnSelector.js

```diff
@@ -268,6 +268,12 @@ export default class ColumnSelector extends Component {
         });
     };
 
+    _handleSort = () => {
+        this.withActualItems(({items}) => {
+            return {items: items.sort((a, b) => a.name.localeCompare(b.name))};
```

Comment:
Default [`itemrRenderer`](https://github.com/ytsaurus/ytsaurus-ui/pull/520/files#diff-b85c197cdef3c085114f9f1a54aade6959549332b2c51af552d118e6be1aa0cfR134) uses `caption` field, so it is a bad idea to sort by `name`  field.
There is two ways to fix it:
1. We need additional property `sortAccessor` near `itemRenderer`, that will be used hear:
```ts
return {items: items.sort(a, b) => sortAccessor(a).localeCompare(sortAccessor(b))};
```
2. Get rid of `itemRenderer` property and mark `caption`  as required property of `items`.

Reply:
corrected. "caption" is now taken into account